test/api/vclamp.cpp

```diff
@@ -58,33 +58,26 @@ int main(void) {
     nrn_segment_diam_set(soma, 0.5, 3);
 
     // voltage clamp at soma(0.5)
-    nrn_double_push(0.5);
-    vclamp = nrn_object_new(nrn_symbol("VClamp"), 1);
+    vclamp = nrn_object_new_d("VClamp", 0.5);
     // 0 mV for 1 ms; 10 mV for the next 2 ms; 5 mV for the next 3 ms
     int i = 0;
     for (auto& [amp, dur]: {std::pair<int, double>{0, 1}, {10, 2}, {5, 3}}) {
         nrn_property_array_set(vclamp, "amp", i, amp);
         nrn_property_array_set(vclamp, "dur", i, dur);
         ++i;
     }
+
     // setup recording
-    v = nrn_object_new(nrn_symbol("Vector"), 0);
-    nrn_rangevar_push(nrn_symbol("v"), soma, 0.5);
-    nrn_double_push(1);
-    nrn_method_call(v, nrn_method_symbol(v, "record"), 2);
```

Comment:
I think this is a case of pushing a double and a range variable without popping them again, i.e. and example of something this PR improves.